queries.sql

```diff
@@ -0,0 +1,27 @@
+/* Querries on the 'animals' table of 'vet_clinic' */
+
+-- Find all animals whoes name ends in "mon"
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+-- List the name of all animals born between 2016 and 2019
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+
+-- List the name of all animals that are neutered and have less than 3 escape attempts
+SELECT name FROM animals WHERE neutered = TRUE AND escape_attempts <= 3;
```

Comment:
- [ ] Kindly you should select the animals that are neutered and have escape attempts less than 3 not less than or equal.

```sql
SELECT name FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
```

Reply:
I totally missed that! Thanks so much!